src/services/content/docs/index.js

```diff
@@ -0,0 +1,77 @@
+module.exports = {
+	contentService: {
+		resourcesService: {
+			description: "A Service for content",
```

Comment:
the resourcesService is not a service for content, but the content service might be... also try to be more descriptive in general.
By the way it's just a proxy Service.